ci/long_running_tests/workloads/serve_handle.py

```diff
@@ -0,0 +1,59 @@
+# A test that stresses the serve handles. We spin up a backend with a bunch
+# of replicas that just forward requests to another backend.
+
+import ray
+from ray import serve
+from ray.serve import BackendConfig
+import logging
+
+num_replicas = 3
+num_queries = 10
```

Comment:
Can we make these parameters using `argparse`?

Reply:
That sounds good.  We'll need to ensure the cluster.yaml has enough workers for all the replicas somehow, so I'll think about how to do that as we integrate with the releaser.

Reply:
For now, I've left it with hardcoded parameters from Simon's example (1, 5, 10 replicas) which seems reasonable to me.